### ISSUE TITLE
pkg.py: pkgs parameter documented as not supported on Windows

### ISSUE THREAD

**Issue by morganwillcock** [saltstack/salt#33313]
The pkgs parameter of pkg.installed and pkg.latest are documented as not supported on Windows:
https://github.com/saltstack/salt/blob/22c433184dbfdaa52846a7b7f64c43c106037771/salt/states/pkg.py#L852-L857
https://github.com/saltstack/salt/blob/22c433184dbfdaa52846a7b7f64c43c106037771/salt/states/pkg.py#L1410-L1416
...but then I realised I was already using them and they seem to work without any problems.
```
install_vcpp_redist:
  pkg.installed:
    - pkgs:
      - microsoft_vcpp_2005_sp1_redist_x64
      - microsoft_vcpp_2005_sp1_redist_x86
      - microsoft_vcpp_2008_redist_x64
      - microsoft_vcpp_2008_redist_x86

install_flash_player:
  pkg.latest:
    - pkgs:
      - adobe_flash_player_activex
      - adobe_flash_player_plugin
```
Tested on 2015.8.8.2

**Comment by Ch3LL** [comment]
@morganwillcock looks like you are right:

```
➜  salt git:(tagv2015.8.1) cat /srv/salt/issues/33313.sls
install_pkgs:
  pkg.installed:
    - pkgs:
      - firefox
      - silverlight
```

```
➜  salt git:(tagv2015.8.1) sudo salt 'win8_test' state.sls issues.33313 -t 30
win8_test:
----------
          ID: install_pkgs
    Function: pkg.installed
      Result: True
     Comment: 2 targeted packages were installed/updated.
     Started: 16:49:35.195000
    Duration: 87852.0 ms
     Changes:   
              ----------
              Mozilla Maintenance Service:
                  ----------
                  new:
                      46.0.1
                  old:
              firefox:
                  ----------
                  new:
                      46.0.1
                  old:
              silverlight:
                  ----------
                  new:
                      5.1.40728.0
                  old:

Summary for win8_test
------------
Succeeded: 1 (changed=1)
Failed:    0
------------
Total states run:     1
```

Looks like we just need to clean up the documentation. Thanks for pointing it out.

**Comment by rallytime** [comment]
Fixed in #33361
